livre/eq/Config.cpp

```diff
@@ -194,16 +199,22 @@ bool Config::init( const int argc, char** argv )
 
 uint32_t Config::frame()
 {
+    if( _impl->dataFrameRange == INVALID_FRAME_RANGE )
+        return eq::Config::finishFrame();
```

Comment:
I just noticed it yesterday that some of my livre_batch instances were stuck not rendering a single frame. I have not debug'd yet though...